src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java

```diff
@@ -173,6 +174,11 @@ private void startFileMetrics(Gaps2File file) {
         fileMetrics.setFileSize(file.getSize());
     }
 
+    @Retryable
```

Comment:
@srochani - Why here vs in SSCS-Common so all services get retry?

Reply:
We have issue only on Case loader at the moment, I think lets try first on CaseLoader if succeed then will put this into sscs-common